auditbeat/Dockerfile

```diff
@@ -10,6 +10,12 @@ RUN \
          iproute2 \
       && rm -rf /var/lib/apt/lists/*
 
+# Use a virtualenv to avoid the PEP668 "externally managed environment" error caused by conflicts
+# with the system Python installation.
```

Comment:
@cmacknz I think we add some more info on why we have added this.

golang:1.19.10 uses debian:12 (bookworm) as base image and golang:1.19.9 uses debian:11 and PEP 668 applies on or above debian:12

What do you think?

Reply:
So that future readers of the code can understand why it was working before on what changed for golang@v1.19.10 image.

Reply:
Sure I can add this, it wasn't obvious to me what was happening without some Googling.